src/components/tables/EntityTable.tsx

```diff
@@ -10,51 +12,208 @@ import {
+                setConnectors(data);
+            }
+        })().catch(() => {});
+
+        (async () => {
+            const { data, error } = await supabaseClient
+                .from(TABLES.LIVE_SPECS)
+                .select(LIVE_SPECS_QUERY)
+                .eq('spec_type', 'capture');
+
+            if (error) {
+                setTableState({
+                    status: TableStatuses.TECHNICAL_DIFFICULTIES,
+                    error,
+                });
+            } else {
+                setPublications(data);
+            }
+        })().catch(() => {});
+    }, [supabaseClient]);
+
+    useEffect(() => {
+        if (publications && connectors) {
+            if (publications.length > 0 && connectors.length > 0) {
+                const formattedPublication: Entity[] = publications.map(
+                    (publication) => {
+                        const catalogNamespace: string =
+                            publication.catalog_name;
+
+                        const dateCreated: string = publication.updated_at;
+
+                        const connectorInfo: ConnectorInfo | undefined =
+                            connectors.find(
+                                (connector) =>
+                                    connector.image_name ===
+                                    publication.connector_image_name
+                            );
+
+                        return {
+                            deploymentStatus: 'ACTIVE',
+                            name: catalogNamespace.substring(
+                                catalogNamespace.lastIndexOf('/') + 1,
+                                catalogNamespace.length
+                            ),
+                            catalogNamespace,
+                            connectorType: connectorInfo
+                                ? connectorInfo.detail
+                                : 'Unknown',
+                            dateCreated,
+                        };
+                    }
+                );
+
+                setUnfilteredEntities(formattedPublication);
+                setTableState({ status: TableStatuses.DATA_FETCHED });
+            } else {
+                setTableState({ status: TableStatuses.NO_EXISTING_DATA });
+            }
+        }
+    }, [publications, connectors]);
+
+    useEffect(() => {
+        const subscription = supabaseClient
+            .from(TABLES.LIVE_SPECS)
+            .on('*', async (payload) => {
+                if (payload.new.spec_type === 'capture') {
+                    setPublications((_publications) =>
+                        _publications
+                            ? [..._publications, payload.new]
+                            : [payload.new]
+                    );
+                }
+            })
+            .subscribe();
+
+        return () => {
+            supabaseClient.removeSubscription(subscription).catch(() => {});
+        };
+    }, [supabaseClient]);
+
+    useEffect(() => {
```

Comment:
The portion of the component state labelled as `entities` is the data root for the component. If filtered entities exist, display that data in the table. Otherwise, display the raw, unfiltered table data.